app/lib/dartdoc/pub_dartdoc_data.dart

```diff
@@ -39,8 +41,14 @@ class ApiElement {
     @required this.documentation,
   });
 
-  factory ApiElement.fromJson(Map<String, dynamic> json) =>
-      _$ApiElementFromJson(json);
+  factory ApiElement.fromJson(Map<String, dynamic> json) {
+    if (json.containsKey('name')) {
```

Comment:
Please move the comment about old data here:
```
   /// Previous data entries may contain
   /// the fully qualified value, but it will be transformed during `fromJson`.
``` 

Instead of having it as part of the doc comment for the `name` property.